docs/src/coupling.md

```diff
@@ -0,0 +1,33 @@
+# [Coupling](@id coupling-id)
+A complex simulation can consist of different spatial domains in which
+different equations are being solved, different numerical methods being used
+or the grid structure is different.
+One example would be a fluid in a tank and an extended hot plate attached to it.
+We would then like to solve the Navier-Stokes equations in the fluid domain
+and the heat conduction equations in the plate.
+The coupling would happen at the interface through the exchange of thermal energy.
+
+Another type of coupling is bulk or volume coupling.
+There we have at least two systems that share all or parts of the domain.
+We could, for instance, have a Maxwell system and a fluid system.
+The coupling would then occur through the Lorentz force.
```

Comment:
That's a nice motivation - but not implemented in Trixi.jl, is it? Maybe it would be good to clarify this?